.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Snyk scan
```

Comment:
Is this related to "Snyk"?

Reply:
Ops! Nope.
Let me change it.

Also, I think there is a syntax error, because I see nothing is running.